scripts/test.sh

```diff
@@ -2,6 +2,37 @@
 set -e
 export RUST_BACKTRACE=full
 
+POCKET_IC_DIR="$(pwd)/target/pocket_ic_test_server"
+export POCKET_IC_BIN="$POCKET_IC_DIR/pocket-ic"
+
+prepare_pocket_ic_server_binary() {
```

Comment:
All this logic before was implemented in Rust in the ic-export crate, this made possible to:
- reuse that code in other projects without duplicating all of this bash code
- transparently execute the tests from within any IDE

Reply:
Will fix

Reply:
Done